src/index.js

```diff
@@ -95,6 +95,30 @@ export default async function fetch(url, options_) {
 			finalize();
 		});
 
+		fixResponseChunkedTransferBadEnding(request_, err => {
+			response.body.destroy(err);
+		});
+
+		/* c8 ignore next 18 */
+		if (process.version < 'v14') {
```

Comment:
This looks very suspicious, and I see https://github.com/node-fetch/node-fetch/issues/1038 discusses a similar thing.

Reply:
You are correct; it takes advantage of string ordering and single-digit node versions are not supported so there is no issue here.